config/config.go

```diff
@@ -37,6 +37,9 @@ type Config struct {
 	// Debug represents to print debug message or not.
 	Debug bool `yaml:"debug"`
 
+	// DisableColorLogging represents if user want to disable colored logging.
+	DisableColorLogging bool `yaml:"disable_log_color"`
```

Comment:
Color logging should be disabled by default, and the default parameter for golang boolean variables is false. Therefore, you need to change this variable to EnableColorLogging.

Reply:
I through that color logging should be enabled by default, which one is better?

Reply:
authorization-proxy made for k8s environment, I think disabled by default is better